### DISCUSSION TITLE
FT8 logging - no log file created

### DISCUSSION THREAD

**Discussion by Dinibaer** [gdyuldin/x6100_gui#52] [Q&A] [answered]
Hello, after the first FT8 qso, there was no logfile.

**Comment by gdyuldin** [comment]
Hi. `ft_log.adi` file on DATA partition creates during starting FT8 app. Also, QSO should be saved to the `qso_log.db` database.
Could you share it, if this file exists?

**Reply by Dinibaer** [reply]
Hi gdyuldin, this file does not exist.

**Reply by gdyuldin** [reply, marked as answer]
Sometimes, DATA partition of the sd card have no letter on windows machines. You can open partition manager and manually assign a letter to the last (3rd) partition of the SD card.

**Reply by Dinibaer** [reply]
That´s it! Many thanks !
